rfc/cart-api-v1.md

```diff
@@ -0,0 +1,179 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+There are two data models required for Cart API: `Cart` and `CartItem`. They are
+designed in the way that allows multiple types of entity in a single `Cart`.
+
+### `Cart`
+| Property | Type | Description | Note |
+| --- | --- | --- | --- |
+| `id` | UUID | Cart ID, auto-generated by the web API | - |
+| `user_id` | UUID | The ID of the user that creates the collection | - |
+| `cart_name` | `str` | Cart Name: If it is `NULL`, the cart is the default one for a corresponding user. | **NEED MORE DISCUSSION** |
+| `default` | `bool` | Flag to indicate whether this cart is the default one for a corresponding user (`False` by default) | **NEED MORE DISCUSSION** |
```

Comment:
FYI default is a reserved word in dynamo so something like `default_cart` or `is_default` should be used.  Any preferences?

Reply:
Let's go with `default_cart`.